<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linear Interpolation Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ME7.jpg</normaloff>ME7.jpg</iconset>
  </property>
  <widget class="QLabel" name="Second_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2번 Property를 입력 (unknown)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x1_LE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>x1값 (table)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_LE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>x값 (problem)</string>
   </property>
  </widget>
  <widget class="QLabel" name="First_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>1번 Property를 입력 (known)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x2_LE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>x2값 (table)</string>
   </property>
  </widget>
  <widget class="QLabel" name="LI_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>linear_interpolation.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_but">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>메인으로</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="x_but">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세제목체</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QLabel" name="x_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>X = 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="x1_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>X1 = 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="x2_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>X2 = 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x1_but">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세제목체</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x2_but">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세제목체</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cal_but">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>100</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>계산하기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Erase_but">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>전체 초기화</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Graph_but">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>그래프 보기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="y1_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Y1 = 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y1_but">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>360</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세제목체</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="y2_LE">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>y2값 (table)</string>
   </property>
  </widget>
  <widget class="QLabel" name="y2_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Y2 = 0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="y1_LE">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>y1값 (table)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y2_but">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>430</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세제목체</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="y_TB">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>510</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>연세소제목체</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'연세소제목체'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;결과&lt;span style=&quot; font-weight:600;&quot;&gt;: &lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Y = None&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <zorder>LI_label</zorder>
  <zorder>Second_label</zorder>
  <zorder>x1_LE</zorder>
  <zorder>x_LE</zorder>
  <zorder>First_label</zorder>
  <zorder>x2_LE</zorder>
  <zorder>Home_but</zorder>
  <zorder>x_but</zorder>
  <zorder>x_label</zorder>
  <zorder>x1_label</zorder>
  <zorder>x2_label</zorder>
  <zorder>x1_but</zorder>
  <zorder>x2_but</zorder>
  <zorder>Cal_but</zorder>
  <zorder>Erase_but</zorder>
  <zorder>Graph_but</zorder>
  <zorder>y1_label</zorder>
  <zorder>y1_but</zorder>
  <zorder>y2_LE</zorder>
  <zorder>y2_label</zorder>
  <zorder>y1_LE</zorder>
  <zorder>y2_but</zorder>
  <zorder>y_TB</zorder>
 </widget>
 <tabstops>
  <tabstop>x1_LE</tabstop>
  <tabstop>x2_LE</tabstop>
  <tabstop>x_LE</tabstop>
  <tabstop>y1_LE</tabstop>
  <tabstop>y2_LE</tabstop>
  <tabstop>Cal_but</tabstop>
  <tabstop>Graph_but</tabstop>
  <tabstop>y_TB</tabstop>
  <tabstop>x1_but</tabstop>
  <tabstop>x2_but</tabstop>
  <tabstop>x_but</tabstop>
  <tabstop>y1_but</tabstop>
  <tabstop>y2_but</tabstop>
  <tabstop>Erase_but</tabstop>
  <tabstop>Home_but</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
